package.json

```diff
@@ -1,12 +1,57 @@
 {
+  "version": "0.4.0",
+  "build": {
+    "productName": "Zazu",
+    "appId": "TinyTacoTeam.ZazuApp",
```

Comment:
`appId` should be `com.tinytacoteam.zazu`, as it will become the identifier of the App,

on `master` branch, the generated `Info.plist` contains:

```xml
        <key>CFBundleIdentifier</key>
        <string>com.tinytacoteam.zazu</string>
```

And in this PR, it become:

```xml
    <key>CFBundleIdentifier</key>
    <string>TinyTacoTeam.ZazuApp</string>
```